specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -300,6 +300,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "post": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "ListResourcePreparationErrors",
```

Comment:
Usually the `operationId` should have a pattern of `PuralNoun_Verb`, therefore could we please change this operationId, for instance `Workspaces_ListResourcePreparationErrors` since this URI is under the URI of a workspace?

Reply:
Can I change it to "ResourcePreparationErrors_List"?